app/views/my_modules/_my_module_details.html.erb

```diff
@@ -1,16 +1,20 @@
 <% my_module_editable = can_manage_module?(@my_module) %>
 
 <div class="flex-block date-block">
-  <span class="fas block-icon fa-calendar-alt"></span>
-  <span class="hidden-xs hidden-sm hidden-md"><%= t('my_modules.module_header.start_date') %></span>
+  <div class="flex-block-label">
+    <span class="fas block-icon fa-calendar-alt"></span>
+    <span class="hidden-xs hidden-sm hidden-md"><%= t('my_modules.details.start_date') %></span>
```

Comment:
I think we need different icons for due date and start date. On small screens, it looks strange.